fiona/_shim.pyx

```diff
@@ -0,0 +1,55 @@
+cdef extern from "ogr_api.h":
```

Comment:
Yes. I shouldn't have added it into the repo. It's a copy of either shim1, shim2 or shim22, in the same way that ogrext.pyx was a copy of ogrext1.pyx/ogrext2.pyx.